metrics/web/lib/debug_menu/data/repositories/hive_local_config_repository.dart

```diff
@@ -0,0 +1,64 @@
+import 'dart:convert';
+
+import 'package:hive/hive.dart';
+import 'package:metrics/common/domain/entities/persistent_store_exception.dart';
+import 'package:metrics/debug_menu/data/model/local_config_data.dart';
+import 'package:metrics/debug_menu/domain/entities/local_config.dart';
+import 'package:metrics/debug_menu/domain/repositories/local_config_repository.dart';
+
+/// An implementation of the [LocalConfigRepository] for [Hive].
+class HiveLocalConfigRepository implements LocalConfigRepository {
+  @override
+  Future<void> open() async {
+    try {
+      await Hive.openBox<String>('local_config');
+    } catch (_) {
+      throw const PersistentStoreException();
```

Comment:
Should we introduce new error codes for the `PersistentStoreException` to indicate the kind of error? Even if in the `readConfig` method this does not matter, soon we'll introduce Sentry logging and it would be great to know about these errors.